src/main/frontend/handler/editor.cljs

```diff
@@ -2002,7 +2002,7 @@
         label (or label "")]
     (case (keyword format)
       :markdown (util/format "![%s](%s)" label link)
-      :org (util/format "[[%s]]"))))
+      :org (util/format "[[%s][%s]]" link label))))
```

Comment:
Org mode doesn't support label directly in the link, check https://orgmode.org/manual/Images.html